sdk/test-utils/recorder/test/sanitizers.spec.ts

```diff
@@ -18,14 +19,14 @@ import { randomUUID } from "@azure/core-util";
     const secretValue = "abcdef";
     let currentValue: string;
 
-    before(() => {
+    beforeAll(() => {
```

Comment:
Why did we have to change this?

Reply:
Because Mocha has `before` and `vitest` has an equivalent as `beforeAll`